packages/e2e/features/ui/components/authenticator/sign-up-federated.feature

```diff
@@ -0,0 +1,11 @@
+Feature: Sign Up with Federation
+
+  Background:
+    Given I'm running the example "ui/components/authenticator/sign-in-federated"
+    When I click "Create account"
+
+  @angular @next @react @vue
+  Scenario: Sign up using OAUTH
+		Then I see the "Google" sign in button
```

Comment:
Weird how this did tabs instead of soft-tab spaces, isn't it?

Reply:
Yeah e2 task: style rules for gherkin.